src/nvim/eval.c

```diff
@@ -17625,7 +17625,7 @@ void ex_function(exarg_T *eap)
                                                (p[2] == 's'))))))
         skip_until = vim_strsave((char_u *)".");
 
-      /* Check for ":python <<EOF", ":tcl <<EOF", etc. */
+      /* Check for ":python <<EOF" */
```

Comment:
With the removal, the comment is too particular, and it serves to confuse. When we add if_lua later we will not remember to update this comment.